run_conformance_tests.py

```diff
@@ -59,6 +59,11 @@ def handle_args(argv):
         help='Disable full conformance mode to avoid extensive tests.',
         required=False,
         action='store_true')
+    parser.add_argument(
+        '--enable-legacy',
+        help='Enable legacy tests.',
+        required=False,
+        action='store_true')
```

Comment:
Shouldn't this also default to true for the conformance report generation?

Reply:
My bad. Fixed both the default value and argument naming by 97f90a65ce88610ee4af1a99d5ff183812b26ae0.